rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -286,7 +286,6 @@ echo " Done"
 # Do this ONLY at first install
 if [ $1 -eq 1 ]
 then
-		echo -e '# This sources the configuration file needed by Rudder\n. /etc/sysconfig/rudder-apache' >> /etc/sysconfig/apache2
```

Comment:
I don't understand what you're saying, sorry. The alternative version below if in a %if statement to make sure it's only on SLES, yes, because /etc/sysconfig/apache2 is SLES-specific.

What do you want me to change?